src/auth_client_demo_assets/vue/App.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+<main>
+  <div v-if="isReady">
+  <LoggedIn v-if="isAuthenticated" />
+  <LoggedOut v-else />
+  </div>
+</main>
+</template>
+
+<script>
+import LoggedOut from './components/LoggedOut.vue'
+import LoggedIn from './components/LoggedIn.vue'
+import { storeToRefs } from 'pinia'
+import { useAuthStore } from "./store/auth";
+
+export default {
+  name: 'App',
+  setup() {
+    const authStore = useAuthStore();
+    const { isAuthenticated } = storeToRefs(authStore);
+    const isReady = isAuthenticated !== null;
```

Comment:
another approach would be to add `isReady` as a [getter](https://pinia.vuejs.org/core-concepts/getters.html) of the `authStore`, this will enable you to simply do `const { isAuthenticated, isReady } = storeToRefs(authStore);` here.